ehitchhiking-ui/src/app/pages/main-screen/routes-list/routes-list.component.ts

```diff
@@ -25,7 +25,7 @@ export class RoutesListComponent implements OnInit {
 
   displayRoute(index: number) {
     this.isChecked = !this.isChecked;
-    if (this.isChecked) {
+    if (this.isChecked && index >= 3 || !this.isChecked && index < 3) {
```

Comment:
(this.isChecked && index >= 3) || (!this.isChecked && index < 3)
3 - magic number

Reply:
done